packages/cfd/src/features/types/containers.types.ts

```diff
@@ -0,0 +1,376 @@
+
+export type TMultiStepRefProps = {
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+export type TInvestorPasswordManager = {
+    toggleModal: () => void;
+    error_message_investor: string;
+    is_submit_success_investor: boolean;
+    setPasswordType: (value: string) => void;
+    onSubmit: (values: TPasswordManagerModalFormValues) => Promise<void>;
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    validatePassword: (values: { old_password: string; new_password: string; password_type: string }) => void | object;
+};
+
+export type TCountdownComponent = {
+    count_from: number;
+    onTimeout: () => void;
+};
+
+export type TCFDPasswordReset = {
+    server: string;
+    account_type: TAccountCategory;
+    password_type: string;
+    account_group: 'demo' | 'real' | '';
+    sendVerifyEmail: () => Promise<VerifyEmailResponse>;
+};
+
+export type TCFDPasswordManagerTabContentWrapper = {
+    steps: Array<{ component: JSX.Element }>;
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+};
+
+export type TCFDPasswordManagerTabContent = {
+    email: string;
+    selected_login: string;
+    platform: TCFDPlatform;
+    toggleModal: () => void;
+    setPasswordType: (value: string) => void;
+    onChangeActiveTabIndex: (value: number) => void;
+    account_group: TCFDPasswordReset['account_group'];
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+};
+
+// cfd-password-manager-modal
+export type TCFDPasswordManagerModal = {
+    is_visible: boolean;
+    platform: TCFDPlatform;
+    selected_login: string;
+    toggleModal: () => void;
+    selected_server: string;
+    selected_account_type: TAccountCategory;
+    selected_account_group: TCFDPasswordReset['account_group'];
+};
+
+// juriscdiction-modal
+export type TJurisdictionCardProps = {
+    disabled: boolean;
+    account_type: TAccountCategory;
+    is_non_idv_design: boolean;
+    account_status: GetAccountStatus;
+    type_of_card: TJurisdictionCardType;
+    jurisdiction_selected_shortcode: string;
+    setJurisdictionSelectedShortcode: (card_type: string) => void;
+};
+
+export type TJurisdictionCardBackProps = {
+    disabled: boolean;
+    card_classname: string;
+    is_card_selected: boolean;
+    toggleCardFlip: React.MouseEventHandler<HTMLSpanElement>;
+    verification_docs: TJurisdictionCardItemVerification | undefined;
+};
+
+export type TJurisdictionCardFrontProps = TJurisdictionCardBackProps & {
+    disabled: boolean;
+    account_status: GetAccountStatus;
+    card_values: TJurisdictionCardItems;
+    type_of_card: TJurisdictionCardType;
+    card_data: TJurisdictionCardSection[];
+};
+
+export type TJurisdictionClickableDescriptionProps = {
+    clickable_description: Array<TClickableDescription>;
+    toggleCardFlip: React.MouseEventHandler<HTMLSpanElement>;
+};
+
+export type TJurisdictionTitleIndicatorProps = {
+    account_status: GetAccountStatus;
```

Comment:
```suggestion
    account_status: GetAccountStatus['status']
```
If im not mistaken this is just the status